### PR TITLE
Add functionality for sending data to backend

### PR DESCRIPTION
**Pull request by camjohnson-code**
## Description

This PR:
- Adds functionality for sending data to the backend. We'll still need to implement the POST function once the be is up and running, but all the pieces are in place for when it's ready.

## What type of PR is this? (check all applicable)

- [X] 💡💫 Feature
- [ ] 🐞🐛 Fix
- [ ] 🪸🎭 Refactor
- [ ] 💅🎨 Style
- [ ] 🧪🔬 Tests
- [ ] 📄💾 Documentation

## Reviewers

Eric Batiste, Cameron Johnson, Jamie Francisco, & Ricky Tran

### CODE REVIEW

**Comment by rickytrandev** [comment]
Hey I pulled down your branch to check out the changes. The data structure might need a little tweaking based on what the BE team sent us with an updated JSON contract. 
This is what the structure should look like: 

```
{
    "start_location_latitude": "40.7128",
    "start_location_longitude": "-74.006",
    "end_location_latitude": "34.0522",
    "end_location_longitude": "-118.2437",
    "trip_duration": "2345234",
    "time_of_trip": "2024-05-13T15:26:03Z",  
    "user_id": 1
}

File: Components/LiveStream.tsx
Comment (ericbatiste) [inline Components/LiveStream.tsx:1]: We should consider building and 'Interface' or 'type' for this component to make the type checking errors go away.

File: App.jsx [hunk 1]
```diff
@@ -7,23 +7,78 @@ import {useState} from 'react';
...
+          )}
         />
         <Stack.Screen
           options={{headerTitle: '', headerShown: false}}
           name="StreamingPage"
-          children={() => <StreamingPage setIsHost={setIsHost} setIsJoined={setIsJoined} isHost={isHost} isJoined={isJoined} />}
+          children={() => (
+            <StreamingPage
+              setTimestamps={setTimestamps}
+              setTimestamps={setTimestamps}
```
Comment (ericbatiste) [inline App.jsx:74]: Looks like you accidentally repeated yourself here.

**Review by ericbatiste — APPROVED** [review]
Looks good for the most part, I left a couple comments but am fine with merging the changes and cleaning it up on the next go round.  Nice work formatting the timestamp!

**Comment by camjohnson-code** [comment]
@rickytrandev @ericbatiste I updated the backendData based on the updated JSON contract

**Pull request merged**